source/application/AMDatamanAppController.cpp

```diff
@@ -356,16 +356,16 @@ bool AMDatamanAppController::startupOnFirstTime()
 			return false;
 	}
 	else{
-		AMFirstTimeWizard ftw;
+		AMFirstTimeWizard *firstTimeWizard = new AMFirstTimeWizard(AMUserSettings::userBasedDataStorage);
 
```

Comment:
@XuetaoLiu-CLS can you add the deleteLater() call?  I think that is the last of any comments.  I'll merge this in once you've done that.